recipe/meta.yaml

```
@@ -19,15 +19,17 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8,<3.10
     - pip
   run:
-    - python >=3.7
-    - dask >=2021.11.1,<=2022.01.0
-    - distributed >=2021.11.1,<=2022.01.0
+    - python >=3.8,<3.10
+    - dask =2022.03.0
+    - distributed =2022.03.0
```

Sorry forgot about the space, but think we still want `==`

```suggestion
    - dask ==2022.03.0
    - distributed ==2022.03.0
```